docs/how-to-guides/internationalization.md

```diff
@@ -87,11 +87,11 @@ This is all you need to make your plugin JavaScript code translatable.
 
 When you set script translations for a handle WordPress will automatically figure out if a translations file exists on translate.wordpress.org, and if so ensure that it's loaded into `wp.i18n` before your script runs. With translate.wordpress.org, plugin authors also do not need to worry about setting up their own infrastructure for translations and can rely on a global community with dozens of active locales. Read more about [WordPress Translations](https://make.wordpress.org/meta/handbook/documentation/translations/).
 
-## Provide Your Own Translations
+## Provide your own translations
 
 You can create and ship your own translations with your plugin, if you have sufficient knowledge of the language(s) you can ensure the translations are available.
 
-### Create Translation File
+### Create atranslation file
```

Comment:
```suggestion
### Create a translation file
```

Or is "the" better here? Because within the same document there is a title called `Load the translation file`.